app/models/concerns/hyrax/solr_document_behavior.rb

```diff
@@ -68,7 +68,8 @@ def collection?
 
     # Method to return the ActiveFedora model
     def hydra_model
-      first(Solrizer.solr_name('has_model', :symbol)).constantize
+      value = first(Solrizer.solr_name('has_model', :symbol))
+      return value.constantize if value
```

Comment:
The original problem case that resulted in this was calling #creator on an empty `SolrDocument.new` would raise, cause it tried to look up the`hydra_model`, which raised cause it had no `has_model_ssim`. 

I think it would be worth considering what you/we _want_ to happen in that case (`SolrDocument.new.creator`), and if various proposed solutions accomplish it. 

I am getting confused thinking about it. If what we want is for `SolrDocument.new.creator` to return empty array/string instead of raising (is that's what's appropriate?)... then I'm not sure which if any of these things accomplishes that. 

If the goal of this is not dealing with that use case... what is the goal of it?

I get confused because it's unclear what the contracts or intentions of many of these methods are, I'm not sure what they are supposed to do.